docs/integrators/deep-history-squad.md

```diff
@@ -103,9 +103,10 @@ Never attach a non-pruned database to a regular observer (i.e. that does not hav
 
 Archives supporting historical lookup are available to download from a Google Cloud Storage bucket.
 
-:::important
-Documentation in this section is preliminary and subject to change.
-:::
+ In order to download archives, you must have [gcloud CLI](https://cloud.google.com/sdk/docs/install) installed and an account on Google Cloud Platform with [billing](https://cloud.google.com/billing/docs/) enabled.
```

Comment:
Rogue leading space.

For billing, is there another resource that is more explicit? For example:
 - https://cloud.google.com/billing/docs/how-to/manage-billing-account
 - https://cloud.google.com/billing/docs/how-to/modify-project

Can we mention the billing requirement in a separate paragraph? Then, we can add both links there.

Reply:
Let's tell users that we've enabled `Request Pays`. Our reason: _in order to avoid unintentional downloads, and promote careful fetching of archives, we've enabled requester pays_. Or something in this fashion.

https://cloud.google.com/storage/docs/requester-pays